hack/local-garden/kubeconfigs/certs/controller-manager.crt

```diff
@@ -0,0 +1,24 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Can you in general create certificates which are valid for a longer period? At the moment it's only a year.

Reply:
Good point, will do that.